CHANGES.rst

```diff
@@ -28,6 +28,9 @@ Features added
 
 * #11328: Mention evaluation of templated content during production of static
   output files.
+* #12704: LaTeX: make :dudir:`contents <table-of-contents>`, :dudir:`topic`,
```

Comment:
I did that [originally](https://github.com/sphinx-doc/sphinx/pull/12704/commits/617b80cab283a494aa6a5e7ca928035efb22a9a5) but then I merged master perhaps more than once.  The problem with leaving it at end, is that it creates conflicts with master each time a new contribution is merged. I think the smoothest work-flow is to avoid merge conflicts and put the burden on @AA-Turner to re-order as desired when releasing.